### ISSUE TITLE
MongoTools only | Setup fails

### ISSUE THREAD

**Issue by maxdewald** [boly38/action-mongo-tools#3]
If you only want to install MongoTools the install fails :
`E: Unable to locate package mongodb-database-tools`

This is because 
```bash
echo "setup keys"
wget -qO - https://www.mongodb.org/static/pgp/server-6.0.asc | sudo apt-key add -
sudo apt-get install gnupg
wget -qO - https://www.mongodb.org/static/pgp/server-6.0.asc | sudo apt-key add -
echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list

echo "aptitude update"
sudo apt-get update
```

only runs in `setupMongoShell.sh`

**Comment by boly38** [comment]
hi @Maggi64 good catch, I'll fix that 👍

**Comment by boly38** [comment]
fixed in [v1.0.1](https://github.com/boly38/action-mongo-tools/releases/tag/v1.0.1)
